roles/openshift_common/meta/main.yml

```diff
@@ -15,4 +15,5 @@ dependencies:
 - { role: os_firewall }
 - { role: openshift_facts }
 - { role: openshift_repos }
+- { role: openshift_docker }
```

Comment:
We don't want to make openshift_docker a dependency on openshift_common, otherwise we'll pull it in for things that aren't dockerized yet.